libs/coin-modules/coin-near/src/js-buildTransaction.ts

```diff
@@ -76,14 +69,23 @@ export const buildTransaction = async (
       actions.push(nearAPI.transactions.transfer(parsedNearAmount));
   }
 
-  const transaction = nearAPI.transactions.createTransaction(
-    a.freshAddress,
-    nearAPI.utils.PublicKey.fromString(publicKey),
-    t.recipient,
-    nonce + 1,
-    actions,
-    nearAPI.utils.serialize.base_decode(block_hash),
-  );
+  try {
+    const transaction = nearAPI.transactions.createTransaction(
+      a.freshAddress,
+      nearAPI.utils.PublicKey.fromString(publicKey),
+      t.recipient,
+      nonce + 1,
+      actions,
+      nearAPI.utils.serialize.base_decode(block_hash),
+    );
 
-  return transaction;
+    return transaction;
+  } catch (e) {
+    log("Near", "Error building transaction", {
+      error: e,
+      transaction: t,
+      account: a,
+    });
```

Comment:
Logging everything necessary if the error [LIVE-7351](https://ledgerhq.atlassian.net/browse/LIVE-7351) isn't fixed with this.
The error is thrown by this [method](https://github.com/near/near-api-js/blob/near-api-js%403.0.2/packages/transactions/src/schema.ts#L35) on `stake` and `transfer` transactions while trying to encode a `u128`. 
Base on that, the only option is that the value provided as `parsedNearAmount` was incorrect and couldn't be encoded properly. See: https://github.com/near/near-api-js/blob/near-api-js%403.0.2/packages/transactions/src/schema.ts#L137-L147